tests/Unit/WPMLTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+class WPMLTest extends WP_UnitTestCase {
+
+	/**
+	 * Tears down the fixture, for example, close a network connection.
+	 *
+	 * This method is called after each test.
+	 *
+	 * @return void
+	 */
+	public function tear_down() {
+		WC_Facebook_WPML_Injector::$settings     = null;
+		WC_Facebook_WPML_Injector::$default_lang = null;
+		remove_all_filters( 'wpml_post_language_details' );
+	}
+
+	public function test_should_hide_product_when_wpml_filter_not_applied() {
+		$this->assertTrue( WC_Facebook_WPML_Injector::should_hide( 1 ) );
```

Comment:
A small suggestion for better readability. 
```suggestion
        $product_id = 1;
        $this->assertTrue( WC_Facebook_WPML_Injector::should_hide( $product_id ) );
```

Reply:
If you agree with this suggestion, you may apply it for the rest of the test cases.

Reply:
Thanks for the suggestion, I did it a little bit differently in 66922dc0.